server/services/random.js

```diff
@@ -33,4 +36,17 @@ module.exports = class RandomService {
         return position;
     }
     
+
+    getResourceScore(radius, x, y){
```

Comment:
Add parameters for min and max resources.

Reply:
@mike-eason  i think it is alright if is as constant